### ISSUE TITLE
Vulnerabilities from jQuery 2.2.4

### ISSUE THREAD

**Issue by ghost** [clippings/layout-grid#21]
Including jQuery 2.2.4 inroduces a potential for XSS vulnerabilities, should be upgraded to minimum of 3.0.0.

I'm happy to upgrade and ensure no jQuery related functionality is broken in the process, but wanted to ensure this project was still being maintained / used first.

**Comment by hkdobrev** [comment]
@jodylecompte Fixing a security issue is always welcome! Could you please send a PR and we could discuss potential BC issues there? Thanks!

**Comment by ghost** [comment]
@hkdobrev Certainly, I'll start digging in later this evening. I'm not familiar off the top of the head with what API changes were made in jQuery to warrant the major version upgrade from 2.X to 3.X, but it's possible the upgrade will be entirely painless.

**Comment by ghost** [comment]
I meant to include the link to the Snky report in my opening comment, that's what initially tipped me off to the problem. 

https://snyk.io/test/npm/jquery/2.2.4
